examples/scaling-criteo/01-Download-Convert.ipynb

```diff
@@ -189,9 +191,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 6,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "distributed.preloading - INFO - Import preload module: dask_cuda.initialize\n"
```

Comment:
did this come from a change you made or is this just a cell you ran and we captured the output by mistake.

Reply:
This are INFO notification, when I initalize the CUDA Cluster

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/19567274/158407758-a8016a86-621e-43c0-b61a-b77f1e3324b0.png">